### DISCUSSION TITLE
Add repository permissions: `Create release`, `Edit release` and `Delete release`

### DISCUSSION THREAD

**Discussion by Zerotask** [community/community#86713] [Repositories]
### Select Topic Area

Product Feedback

### Body

As a repository owner I want to disallow members of a specific group to create, edit or delete releases.
I'd like to have those new permissions be assigned to the `Pre-defined roles` `Maintain` and `Admin`. This way, people with just the `Write` role can see release, but they can't create, edit or delete them.

**Comment by tstellar** [comment]
In the wake of the xz compromise, this feature is very important.  Especially for large repositories that have a lot of committers.
